### PR TITLE
Ability to download capstone git as zip

### PR DESCRIPTION
**Pull request by XVilka**
To solve the problem of building without `git clone`.

https://github.com/radare/radare2/issues/10152

@radare please check this one. It activated like `CS_COMMIT_ARCHIVE=1 make`

### CODE REVIEW

File: shlr/capstone.sh [hunk 1]
```diff
@@ -32,11 +33,27 @@ parse_capstone_tip() {
 	BRANCH="$(git rev-parse --abbrev-ref HEAD)"
 }
 
-clone_capstone() {
+download_archive() {
+	echo '[capstone] Downloading capstone snapshot...' >&2
+	wget -O .cs_tmp.zip "$CS_ARCHIVE" || fatal_msg 'Cannot download archived capstone'
```
Comment (radare) [inline shlr/capstone.sh]: wget is being removed  from many operating systems, we should support curl too

**Comment by XVilka** [comment]
@radare what do you mean by "an option"?

**Comment by radare** [comment]
A way for the user to choose

**Comment by XVilka** [comment]
@radare - user can choose by setting the environment variable:
```bash
CS_COMMIT_ARCHIVE=1 make
```

**Pull request merged**